### ISSUE TITLE
python/django/security/django-no-csrf-token.yaml incorrect regexp

### ISSUE THREAD

**Issue by minusf** [returntocorp/sgrep-rules#3638]
**Describe the bug**
When the `csrf_token` follows the `<form>` right away, a false positive is generated.

**To Reproduce**
```
<form method="post">{% csrf_token %}
```

**Expected behavior**
Rule passes.

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [x] P2: annoying but not blocking me

**Additional Context**
